README.md

```diff
@@ -33,3 +33,37 @@ Example:
   "head_sha": "9130a5840e1ac7ffe198795ba663fdd545b390c3"
 }
 ```
+
+## Staging rollout procedure
+
+To rollout to staging, we want to first make staging be identical with prod. Thus, we clone the prod db to staging, then restart the backend on the cloned db:
+
+```sh
+ssh bastion "cd pathoplexus/scripts/db-clone/ && ./clone-prod-to-staging.sh"
+kubectl rollout restart deployment/loculus-backend -n staging
+```
+
+We then make sure that staging really is identical to prod by running the integrity script from `pathoplexus/pathoplexus`:
+
+```sh
+cd ~/code/pathoplexus/data-integrity-tests/regression-testing
+micromamba activate pp-integrity
+snakemake -F
+cat  results/*.diff
+```
+
+The result is expected to be just empty lines.
+
+We can then rollout a new commit to staging, via PR in this repo:
+
+```sh
+gh workflow run bump-staging.yml -R pathoplexus/loculus_deployments
```

Comment:
I think this is also a command specific to you

Reply:
don't think so :)